src/main/java/seatsio/util/UnirestUtil.java

```diff
@@ -32,4 +36,25 @@
             throw new SeatsioException(e, request.getHttpRequest());
         }
     }
+
+    public static GetRequest get(String url, String secretKey, Long accountId) {
+        return authenticate(Unirest.get(url), secretKey, accountId);
+    }
+
+    public static HttpRequestWithBody post(String url, String secretKey, Long accountId) {
+        return authenticate(Unirest.post(url), secretKey, accountId);
```

Comment:
UnirestUtil ?

Reply:
The class `UnirestUtil` uses `Unirest` under the hood